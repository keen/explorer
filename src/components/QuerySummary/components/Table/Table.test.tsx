import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';

import StyledTable from './index';

test('should render <StyledTable/>', () => {
  const { container } = render(
    <StyledTable.Table>
      <StyledTable.Body>
        <StyledTable.Row>
          <StyledTable.Label>Label</StyledTable.Label>
          <StyledTable.Value>Value</StyledTable.Value>
        </StyledTable.Row>
      </StyledTable.Body>
    </StyledTable.Table>
  );

  expect(container).toMatchSnapshot();
});
