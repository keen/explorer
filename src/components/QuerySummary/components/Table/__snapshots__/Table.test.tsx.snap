// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <StyledTable/> 1`] = `
.c0 {
  margin: 0 0 10px 0;
  padding: 0;
  border-collapse: collapse;
  table-layout: fixed;
  text-align: left;
}

.c3 {
  padding: 0 20px 10px 0;
  font-family: 'Lato Bold',sans-serif;
  font-size: 14px;
  line-height: 17px;
  color: #4F5B5F;
}

.c5 {
  padding: 0 0 10px 0;
  font-family: 'Lato Regular',sans-serif;
  font-size: 14px;
  line-height: 17px;
  color: #4F5B5F;
}

.c1 {
  vertical-align: top;
}

.c1:last-child .c2,
.c1:last-child .c4 {
  padding-bottom: 0;
}

<div>
  <table
    class="c0"
  >
    <tbody
      class=""
    >
      <tr
        class="c1"
      >
        <th
          class="c2 c3"
        >
          Label
        </th>
        <td
          class="c4 c5"
        >
          Value
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
